uk.co.compucorp.civicrm.hremails/info.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<extension key="uk.co.compucorp.civicrm.hremails" type="module">
+  <file>hremails</file>
+  <name>CiviHR Emails</name>
+  <description>Stores and handles CiviHR's email templates</description>
+  <license>AGPL-3.0</license>
+  <maintainer>
+    <author>Alessandro Verdura</author>
+    <email>alessandro@compucorp.co.uk</email>
+  </maintainer>
+  <releaseDate>2017-04-26</releaseDate>
+  <version>1.0</version>
```

Comment:
Since we'll keep all the extensions in the same version for now, this should be 1.6.9